audit/projects/k8s-artifacts-prod/services/compute/project-info.json

```diff
@@ -62,7 +62,7 @@
       "metric": "URL_MAPS"
     },
     {
-      "limit": 30,
+      "limit": 150,
```

Comment:
OK I'm beginning to wonder if we need to do something about this. It feels like most of the PRs we've seen this week have looked something like this